src/components/CreateAddress/PublicKeyImporter.jsx

```diff
@@ -128,59 +129,50 @@ class PublicKeyImporter extends React.Component {
   };
 
   renderImportByMethod = () => {
-    const {
-      publicKeyImporter,
-      network,
-      addressType,
-      defaultBIP32Path,
-    } = this.props;
+    const { publicKeyImporter, network, defaultBIP32Path } = this.props;
+
     if (
       publicKeyImporter.method === TREZOR ||
       publicKeyImporter.method === LEDGER
     ) {
       return (
         <HardwareWalletPublicKeyImporter
-          publicKeyImporter={publicKeyImporter}
-          validateAndSetPublicKey={this.validateAndSetPublicKey}
-          validateAndSetBIP32Path={this.validateAndSetBIP32Path}
-          resetBIP32Path={this.resetBIP32Path}
+          method={publicKeyImporter.method}
+          validatePublicKey={this.validatePublicKey}
           enableChangeMethod={this.enableChangeMethod}
           disableChangeMethod={this.disableChangeMethod}
-          addressType={addressType}
           defaultBIP32Path={defaultBIP32Path}
           network={network}
+          onImport={this.handleImport}
         />
       );
     }
     if (publicKeyImporter.method === HERMIT) {
       return (
         <HermitPublicKeyImporter
-          publicKeyImporter={publicKeyImporter}
-          validateAndSetPublicKey={this.validateAndSetPublicKey}
-          validateAndSetBIP32Path={this.validateAndSetBIP32Path}
-          resetBIP32Path={this.resetBIP32Path}
+          network={network}
```

Comment:
Wow, it looks like `network` was never getting passed before to `HermitPublicKeyImporter` even though the component uses it. Good find.